src/integrationTest/java/org/opensearch/node/PluginAwareNode.java

```diff
@@ -26,7 +26,7 @@
 
 package org.opensearch.node;
 
-import java.util.Arrays;
```

Comment:
Is the refactor part of this change or just tangential? Does something about the arrays prevent the warnings?

Reply:
I replaced it with `List.of()`